src/stdune/ordered.ml

```diff
@@ -0,0 +1,5 @@
+module Make (Key : Map.Key) = struct
```

Comment:
In base this is called `Comparable.Make`. I think we should try to use an existing name if we can.